src/mapblock.h

```diff
@@ -310,17 +318,17 @@ class MapBlock
 	//// Caller must ensure that this is not a dummy block (by calling isDummy())
 	////
 
-	inline const MapNode &getNodeUnsafe(s16 x, s16 y, s16 z)
+	inline MapNode getNodeUnsafe(s16 x, s16 y, s16 z) const noexcept
```

Comment:
how large is a `MapNode`, and how often is it called to take a const& return value?

Reply:
> how large is a MapNode

4 bytes.